packages/cojson/src/tests/sync.test.ts

```diff
@@ -33,454 +40,6 @@ beforeEach(async () => {
   jazzCloud = setupSyncServer();
 });
 
-test("Node replies with initial tx and header to empty subscribe", async () => {
```

Comment:
Replaced these tests with the ones inside `sync.load` and `sync.peerReconciliation`